pds_doi_core/cmd/DOIWebClient.py

```diff
@@ -0,0 +1,284 @@
+#!/bin/python
```

Comment:
This function must be integrated with the argparse general processing:
- With subparser 'release' for doi in draft status. (can be done later)
- Directly at the end of the 'reserve' process. The function should be in the output module.

Reply:
The file DOIWebClient.py has been replaced with pds_doi_client.py in cmd sub directory.

It is used by the main cmd/pds_doi_cmd.py module.